### ISSUE TITLE
Application error when tabbing away from dropdowns

### ISSUE THREAD

**Issue by ndarilek** [podverse/podverse-web#1040]
**Describe the bug**
I get an application error whenever tabbing away from and back to dropdowns, with the following console error:

TypeError: can't access property "focus", this.item.current is null
    NextJS 14
framework-5f4595e5518b5600.js:1:82741
TypeError: can't access property "focus", this.item.current is null
    NextJS 14
main-d977f1d2acb21ba7.js:1:11705
A client-side exception has occurred, see here for more info: https://nextjs.org/docs/messages/client-side-exception-occurred main-d977f1d2acb21ba7.js:1:11722

**To Reproduce**

1. Go to https://podverse.fm.
2. Find one of the dropdowns. I'm a screen reader user, so these announce that you can press enter on them and tab/arrow away.
3. Press enter.
4. Press tab. This seems to throw focus completely outside the browser window.
5. I don't know if shift-tabbing back into the window is required to make the error appear, but I have to do this to notice it with the screen reader.

**Expected behavior**
I should be able to tab to the next control.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Firefox nightly
 - Version Latest, all updates applied

**Additional context**
I'd suggest using the [listbox pattern](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/listbox_role) for this if you can. In particular, all the explanatory "press down arrow or tab or..." is a bit wordy, and you can probably skip that entirely if you follow the keyboard navigation patterns as described at the previous link.

Thanks!

**Comment by mitchdowney** [comment]
We are archiving all Github issues that pertain to the previous generation Podverse. The Podverse dev team is currently 100% focused on the next generation app, and will be only working on next gen Github issues going forward.

If you would like to see a work-in-progress demo of the upcoming Podverse website, please visit https://alpha.podverse.fm. Our tentative release date is Spring 2026.

If you're a developer who may be interested in contributing to Podverse Next-Gen, please see our [Local Dev Setup documentation](https://github.com/podverse/podverse-ops/blob/v5-develop/dev/local-dev-setup.md). We can also be reached through [Matrix or Discord](https://podverse.fm/contact).

I'm sorry for the abrupt announcement, but thank you for your bug reports and feature requests.
